src/consensus/vertex.cpp

```diff
@@ -65,6 +66,11 @@ void Vertex::UpdateMilestoneReward() {
     cumulativeReward += GetParams().reward * ((snapshot->GetLevelSet().size() - 1) / GetParams().msRewardCoefficient);
 }
 
+size_t Vertex::GetTxNum() const {
+    return std::accumulate(validity.cbegin(), validity.cend(), 0,
+                           [](const size_t& sum, const uint8_t& v) { return sum + (v & Vertex::Validity::VALID); });
```

Comment:
For `sum + (v & Vertex::Validity::VALID)`, if we won't change the value of `Validity::VALID`(1), we should just use  `sum + v`, and if we may change the value (maybe set it to be 2), then we should use `sum + (v == Vertex::Validity::VALID)` because `2 & 2 = 2  != 1`.